src/Sandbox/restart.cpp

```diff
@@ -253,10 +261,11 @@ int main(int argc, char** argv)
   elecs[0].destroyWalkers(elecs[0].begin(), elecs[0].end());
 
   // load walkers
-  const char* restart_input = "<tmp> \
-  <mcwalkerset fileroot=\"restart\" node=\"-1\" version=\"3 0\" collected=\"yes\"/> \
+  std::string restart_input_directory = "<tmp> \
+  <mcwalkerset fileroot=\"" + directory + "restart\" node=\"-1\" version=\"3 0\" collected=\"yes\"/> \
```

Comment:
We are moving to string literals.
Since you exposed this line of code, could you remove all the backslash. See change examples.
https://github.com/QMCPACK/qmcpack/pull/4303/files

Reply:
Got it! Done